netdata-installer.sh

```diff
@@ -45,14 +45,14 @@ download() {
 	dest="${2}"
 	if command -v wget >/dev/null 2>&1; then
 		if [ -t 1 ]; then
-			run wget -O - "${url}" >"${dest}" || fatal "Cannot download ${url}"
+			run wget --timeout=5 --tries=3 -O - "${url}" >"${dest}"
 		else
-			run wget --progress=dot:mega -O - "${url}" >"${dest}" || fatal "Cannot download ${url}"
+			run wget --timeout=5 --tries=3 --progress=dot:mega -O - "${url}" >"${dest}"
 		fi
 	elif command -v curl >/dev/null 2>&1; then
-		run curl -L "${url}" >"${dest}" || fatal "Cannot download ${url}"
+		run curl -L --connect-timeout 10 --retry 3 "${url}" >"${dest}"
 	else
-		fatal "I need curl or wget to proceed, but neither is available on this system."
+		echo >&2 "I need curl or wget to proceed, but neither is available on this system."
```

Comment:
Why was this changed? By default end user is supposed to install go.d.plugin, if anything during this installation fails, then installer should fail. That's the same as if sth in python.d.plugin installation. If someone doesn't want `go.d.plugin` to be installed, then flag `--disable-go` should be used.

Reply:
If there was a problem with this part due to lack of `fatal()` or sth like that, then it should be reintroduced as:
```
fatal() {
	printf >&2 "${TPUT_BGRED}${TPUT_WHITE}${TPUT_BOLD} ABORTED ${TPUT_RESET} ${*} \n\n"
	exit 1
}
```